dor/entrypoints/api/catalog.py

```diff
@@ -0,0 +1,51 @@
+import uuid
+
+from dor.config import config
+from dor.service_layer import catalog_service
+import sqlalchemy
+from fastapi import APIRouter, Depends, status
+from fastapi.responses import JSONResponse
+
+from dor.adapters.catalog import SqlalchemyCatalog
+
+
+catalog_router = APIRouter(prefix="/catalog")
+
+ENGINE = sqlalchemy.create_engine(config.get_database_engine_url())
+
+
+def get_db_session():
+    with sqlalchemy.orm.Session(ENGINE) as session:
+        yield session
+
+
+@catalog_router.get("/bins/{identifier}/")
+def get_bin_summary(identifier: str, session = Depends(get_db_session)) -> JSONResponse:
+    try:
+        uuid_identifier = uuid.UUID(identifier)
+    except ValueError:
+        return JSONResponse(status_code=status.HTTP_400_BAD_REQUEST, content="Identifier is not a valid UUID.")
```

Comment:
I don't currently have tests for the `400` or `404` failure cases. Should I add them?

Reply:
Nah. We'll get to them as the API unfolds.